lua/buftabline/commands.lua

```diff
@@ -56,22 +56,11 @@ M.on_buffer_delete = b.on_buffer_delete
 M.on_tab_closed = b.on_tab_closed
 M.on_vim_enter = b.on_vim_enter
 
-M.build = function()
-    vim.schedule(function()
-        local _, err = xpcall(require("buftabline.build"), debug.traceback)
-        if not err then
-            return
-        end
-
-        u.echo_warning("Something went wrong!: " .. err)
-        vim.o.tabline = ""
-        u.clear_augroup()
-    end)
-end
+M.build = require("buftabline.build")
```

Comment:
Probably don't need this anymore, and can just use `require("buftabline.build")()` directly in the global function defined in init.lua. I can make this so if you wish.

Reply:
Agreed that there's no need for this and we can delete it.